.pre-commit-config.yaml

```diff
@@ -91,3 +91,10 @@ repos:
     - requests_unixsocket
     - six
     - Sphinx
+
```

Comment:
plz follow the same style as the rest of the file — with a single line separator.
```suggestion
```